src/pages/get-eth.tsx

```diff
@@ -229,7 +229,7 @@ const GetETHPage = ({ data }: PageProps<Queries.GetEthPageQuery>) => {
         "page-get-eth-article-keeping-crypto-safe",
         intl
       ),
-      link: "https://blog.coinbase.com/the-keys-to-keeping-your-crypto-safe-96d497cce6cf",
+      link: "https://medium.com/the-coinbase-blog/the-keys-to-keeping-your-crypto-safe-96d497cce6cf",
```

Comment:
For me, this just redirects to https://www.coinbase.com/blog/the-keys-to-keeping-your-crypto-safe-96d497cce6cf (the 404) 🤔

Reply:
I found this version on wayback machine: https://web.archive.org/web/20190716160333/https://blog.coinbase.com/the-keys-to-keeping-your-crypto-safe-96d497cce6cf?gi=548619266f28

I think we'll probably want to replace this with something else, but for now, let's use that version so we're not 404ing at the very least.

```suggestion
      link: "https://web.archive.org/web/20190716160333/https://blog.coinbase.com/the-keys-to-keeping-your-crypto-safe-96d497cce6cf?gi=548619266f28",
```

Reply:
Yes, when I rechecked it also

> For me, this just redirects to https://www.coinbase.com/blog/the-keys-to-keeping-your-crypto-safe-96d497cce6cf (the 404) 🤔

When I rechecked it also shows the 404 again. I must have viewed the medium article in waybacke machine somehow. Approved your edit.